Production Release

jkeasley opened the pull request:
This PR is a production release for the NHSX site, which should deploy the changes made to amend the IG header, as well as the additional support for GA4 analytics requested by the NHSX team

dragon-dxw committed:
Migrate blocks in preparation for nhsuk-frontend 1.0 upgrade

Transformation documentation:
https://github.com/nhsuk/wagtail-nhsuk-frontend/blob/master/CHANGELOG.md#v100
With great thanks to Andy Babic for explaining how to migrate
https://wagtailcms.slack.com/archives/C81FGJR2S/p165874812380607

dragon-dxw committed:
Add all other non-abstract page types that have a body element

dragon-dxw committed:
Non-deprecated poetry install

dragon-dxw committed:
Merge pull request #724 from nhsx/migrate-panels

Migrate blocks in preparation for nhsuk-frontend 1.0 upgrade

dragon-dxw committed:
Update Poetry to 1.2.2

dragon-dxw committed:
Check migrations exist before committing

dragon-dxw committed:
Merge pull request #738 from nhsx/poetry-1.2.2

Update Poetry to 1.2.2

dragon-dxw committed:
Merge pull request #739 from nhsx/precommit-migration-checker

Check migrations exist before committing

fultonesser committed:
Change email listed in IG header

fultonesser committed:
Merge pull request #740 from nhsx/fix/17114-update-ig-header

Change email listed in IG header

dragon-dxw committed:
Remove breakpoints and some prints from Wagtail Frontend 1.0 migration

dragon-dxw committed:
Bump Black to 22.10 and Python to 3.7

dragon-dxw committed:
Merge pull request #743 from nhsx/migration-remove-breakpoints

Remove breakpoints and some prints from Wagtail Frontend 1.0 migration

jkeasley committed:
Add GA4 analytics to the NHSX site

This commit adds GA4 as an option to the analytics settings, with
a new field for the GA4 ID in the Analytics setting page, and a new
section in the base template which will include the GA4 snippet with
the configured ID if it has been set to a value

https://dxw.zendesk.com/agent/tickets/17221

jkeasley committed:
formatting fix

jkeasley committed:
Merge pull request #745 from nhsx/feature/17221-ga4-implemented-with-admin-config

Add GA4 analytics to the NHSX site

jkeasley committed:
Modify migration dependency

This commit modifies the migration dependency so that it depends on
0047_nhsuk_frontend_panels rather than 0046_auto_20210902_1250,
this is to resolve a conflict whjen running the migration on the stage
instance

jkeasley committed:
Merge pull request #748 from nhsx/feature/17221-ga4-implemented-with-admin-config

Modify migration dependency

dragon-dxw committed:
Update README.md

dragon-dxw reviewed (changes requested):
We've since  had a quick chat about how we're going to need to change the data migration to do nothing because it's not quite right: that ones on me, sorry!

jkeasley committed:
disable frontend migration migration

This commit idabled migration 0047_nhsuk_frontend_panels.py so that
it is not executed when applying migrations, so that the currently
dev branch can be deployed to prod safely.

jkeasley committed:
Merge pull request #755 from nhsx/chore/disable-upgrade-migration

disable frontend migration migration
